### DISCUSSION TITLE
Code security and secret scanning insights for your enterprise (public beta)

### DISCUSSION THREAD

**Discussion by goldieowner17** [community/community#125183] [Enterprise]
Gain valuable insights and effectively monitor your enterprise’s security landscape and progress with two new enterprise-level pages: the security overview dashboard and secret scanning metrics.

![enterprise-scanning-insights](https://github.com/community/community/assets/166861992/6da46ca2-caff-4929-98f2-0924f1d98f93)

New overview dashboard on the security tab at the organization level

Key features:
- Customizable filters: Select specific time periods and focus areas such as security tool, team, or custom repository property.
- Comprehensive data: Trending and snapshot data provide a robust security landscape overview.
- Detailed metrics: Includes metrics such as the average age of security alerts, mean time to remediate, and push protection statistics.

**Comment by PhyoPhyo602** [comment]
Facebook&Twitter
